server/routerlicious/packages/lambdas/src/nexus/index.ts

```diff
@@ -138,18 +138,30 @@ export function configureWebSocketServices(
 	webSocketServer.on("connection", (socket: core.IWebSocket) => {
 		// Timer to check token expiry for this socket connection
 		const expirationTimer = new ExpirationTimer(() => socket.disconnect(true));
-		// Map from client IDs on this connection to the object ID and user info.
+
+		/**
+		 * Maps and sets to track various information related to client connections.
+		 * Note: These maps are expected to have only one entry per client ID.
```

Comment:
This still sounds like there are more than one client id. The expectation is that there will be only one client id. `createMessageClientAndJoinRoom` will only be called once per socket connection. There is one `connectDocument`.

Reply:
whoops, yeah silly mistake